File: kattis/easy/fiftyshades.java
package easy;

import java.io.BufferedReader;
import java.io.IOException;
import java.io.InputStreamReader;

public class fiftyshades {
	public static void main(String[] args) throws IOException {
		try (BufferedReader br = new BufferedReader(new InputStreamReader(System.in))) {
			int n = Integer.parseInt(br.readLine());

			int attend = 0;
			for (int i=0; i < n; i++) {
				String line = br.readLine().toLowerCase();
				if (line.contains("rose") || line.contains("pink"))
					attend++;
			}
			System.out.println(attend > 0 ? attend : "I must watch Star Wars with my daughter");
		}
	}
}

File: kattis/easy/ccpp/fiftyshades.c
//
// Created by BBJ on 12.01.2024.
//
#include <stdio.h>
#include <string.h>
#include <ctype.h>

int main() {
    int n;
    scanf("%d", &n);

    int attend = 0;
    for (int i = 0; i < n; ++i) {

        // getting line with buffer and scanf
        char line[31];
        scanf("%30s", line);

        // lowering line manually
        for (int j = 0; line[j]; ++j) {
            line[j] = (char) tolower(line[j]);
        }

        // checking for keywords using substring
        if (strstr(line, "rose") != NULL || strstr(line, "pink") != NULL)
            attend++;
    }

    if (attend > 0) {
        printf("%d\n", attend);
    } else {
        printf("I must watch Star Wars with my daughter\n");
    }

    return 0;
}